Source/WebCore/dom/Document.h

```diff
@@ -1928,6 +1931,7 @@ class Document
     bool m_hasXMLDeclaration { false };
 
     AtomString m_contentLanguage;
+    AtomString m_effectiveDocumentElementLanguage; // DocumentElement's lang attribute value.
```

Comment:
The variable name could be just m_documentElementLanguage as it is the actual value rather than an effective value. The accessor however returns an effective value (and is correctly named) because it falls back to m_contentLanguage.

Reply:
I guess setter should also be setDocumentElementLanguage.